### ISSUE TITLE
Din't receive any call from the sample Multiphase app

### ISSUE THREAD

**Issue by st-small** [oliverepper/libphone#7]
Hi folks!
Have installed all dependencies through the script (`build-darwin-dependencies`). Trying to make a call in the Multiphone example app with my credentials, but receive only message in the Viber application that I missed a call. Is any solutions for this behaviour?

Thanks in advance.

**Comment by oliverepper** [comment]
Hi. Can you provide a bigger part of the log, please?

**Comment by st-small** [comment]
@oliverepper yes, sure. But I should to replaced account and sip provider values with `<my-sip-account>` and `<my-sip-provider>` placeholders as it requires my NDA. In my tests there was used real values.

[Multiphone investigation log.txt](https://github.com/user-attachments/files/16797007/Multiphone.investigation.log.txt)
